README.md

```diff
@@ -96,8 +99,9 @@ Since both C2DM and GCM are still available, the `AndroidMessage` class has a sm
 
     $message = new AndroidMessage();
     $message->setGCM(true);
+    $message->setFCM(true); // Use to Firebase Cloud Messaging
 
-to send as a GCM message rather than C2DM.
+to send as a FCM message rather than GCM or C2DM.
```

Comment:
This is slightly unclear I think, as you should be specifying only one of GCM or FCM - you don't need to specify both. Could you reword this so that it's clear you only need to use one of the `set*()` methods, depending on your use case?